infrastructure/api-configuration/dev-secrets

```diff
@@ -1,2 +1,5 @@
 DJANGO_SECRET_KEY=THIS_IS_NOT_A_SECRET_DO_NOT_USE_IN_PROD
 DATABASE_PASSWORD=THIS_IS_NOT_A_SECRET_DO_NOT_USE_IN_PROD
+
+OAUTH_CLIENT_SECRET=1aaa0a94-5605-4cd8-95a4-6724c2f7fd42
+OAUTH_URL=https://sandbox.orcid.org/oauth/token
```

Comment:
Will this need to be changed manually off of sandbox when we go to full deployment?

Reply:
Ah- never mind. Just read below.